modules/error.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+
+app.use(errorHandler); // Error Middleware

```

Comment:
These lines all need to go. Instead, put `app.use(errorHandler)` back in your `server.js`.

What's happening here is you're making a new `app` that's never started (with `app.listen()`), but at least it will have handled errors.